rust/private/rustc.bzl

```diff
@@ -478,9 +478,9 @@ def _process_build_scripts(
             - (File): An optional path to a generated environment file from a `cargo_build_script` target
             - (depset[File]): All direct and transitive build flags from the current build info.
     """
-    extra_inputs, out_dir, build_env_file, build_flags_files = _create_extra_input_args(build_info, dep_info)
+    extra_inputs, out_dir, build_env_file, flags_files, linker_flags_files = _create_extra_input_args(build_info, dep_info)
```

Comment:
nit: I kinda liked the `build_*` prefix to indicate what came from a `BuildInfo`